addons/docs/README.md

```diff
@@ -4,6 +4,8 @@
 
 # Storybook Docs
 
+> migration guide: This page document the method to configure storybook introduced recently in 5.3.0, consult the [migration guide](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md) if you want to migrate to this format of configuring storybook.
```

Comment:
This page document => This page documents (across the board)

Reply:
✔️